appservice/src/startStreamingLogs.ts

```diff
@@ -61,8 +61,10 @@ export async function startStreamingLogs(client: SiteClient, verifyLoggingEnable
                 }
 
                 await new Promise((onLogStreamEnded: () => void, reject: (err: Error) => void): void => {
+                    let recentData: string = '';
                     let newLogStream: ILogStream;
                     const logsRequest: request.Request = requestApi(`${client.kuduUrl}/api/logstream/${logsPath}`);
+                    const recentDataTimer: NodeJS.Timer = setInterval(() => { recentData = ''; }, 2 * 1000);
                     newLogStream = {
```

Comment:
Where is newLogStream used?

Reply:
Uh a few places in this function. It's actually the return value of this root `startStreamingLogs` function. 

`startStreamingLogs` is admittedly a confusing mess of promises which could possibly be improved, but part of that is just the nature of a long-running action like log streaming